typescript/sdk/src/ism/EvmIsmModule.hardhat-test.ts

```diff
@@ -429,5 +505,99 @@ describe('EvmIsmModule', async () => {
           .true;
       });
     }
+
+    it(`reordering modules in an existing staticAggregationIsm should not trigger a redeployment`, async () => {
```

Comment:
@paulbalaji - should we expect this test to pass? it is not atm. I think this is a limitation of the sdk where a change in the order of the modules of the `staticAggregationIsm` with lead to redeployment.

One thing we can to do to mitigate this is sort the config by sorting the domains array by the ism type. This will work if there are not duplicate types an ism in the `staticAggregationIsm`. This should cover most cases

Reply:
@paulbalaji - can you look at this test! Should it pass in your opinion?